indico/modules/attachments/__init__.py

```diff
@@ -33,6 +33,25 @@
 
 connect_legacy_signals()
 
+material_types = [
```

Comment:
`MATERIAL_TYPES`, since it's a constant? But maybe we should rename it to `DEFAULT_FOLDER_NAMES`, now that we don't user "material" in the code?